### ISSUE TITLE
Bullet spread as a factor of balance

### ISSUE THREAD

**Issue by drNovikov** [zrisher/GardenWarfare#11]
In order to make turrets less defenseless against long-range sniping suggest giving player-controlled weapons a cone of fire.

I've been testing different mods, including the Gatling Cannon mod by Ravenbolt, and have found out that it is possible. Bullets form a cone of fire, while a small portion of them remains "precise" and follows the same direct path to where the gun is aiming:

![Image of bullet spread testing](http://i.imgur.com/oNnjKBk.jpg)

The spread of this weapon is probably too wide though.

I think people should be able to kill turrets with rapid firing small ship cannons at 300-400 meters, but at 500-800 meters cone of fire should make it economically unreasonable. 300-400 meters is a range where they need to move really fast to avoid incoming fire, and probably switch to missiles.

**Comment by zrisher** [comment]
Well I definitely agree that bullet spread should be more prevalent and have a greater influence on tactics. But I think the cone size should be highly dependent on the type of weapon and ammo.

I don't think weapons should be more or less accurate depending on if a player is controlling them. Doesn't make much sense to me.

The problem you're trying to solve - turret sniping - is definitely an issue, but I think that our plans to have a turret with equal range for every static weapon should fix that. And generally platforms and big ships can have longer-ranged, more powerful weapons because they're just heavier and their internals are bigger.

**Comment by drNovikov** [comment]
I absolutely agree that the spread should depend on weapon and ammo.

Speaking of the sniping problem, the equal range is not enough. Consider a player staying at the very edge of the danger zone and moving in-out for shooting. The range is big enough to provide a significant bullet travel time, so the player can move in, aim, shoot, and move out before bullets hit him. I'd suggest an additional mechanic that gives advantage to stationary and massive ships/stations. When a player quickly moves in and out of the danger zone, he should not retain the same highest level of accuracy (or CoF angle). It should take some time for the spread to tighten. Stationary or non-accelerating ships should be more accurate. The heavier the ship is, the less CoF bloom should be.

I believe the autopilot mod with some pre-programmed drone behaviors (patrol, attack, pursuit, etc) could help. Maybe even pre-made drones, like those in EVE Online.

**Comment by zrisher** [comment]
Yeah I agree - having larger spread while and immediately after moving would make a lot of sense.

**Comment by drNovikov** [comment]
Probably, not the movement itself, but acceleration/deceleration, linear and angular.
